src/components/MessagesList/MessagesList.vue

```diff
@@ -1106,66 +1094,35 @@ export default {
 		},
 
 		/**
-		 * @param {object} options Event options
-		 * @param {boolean} options.force Set to true, if the chat should be scrolled to the bottom even when it was not before
-		 */
-		handleScrollChatToBottomEvent(options) {
-			if ((options && options.force) || this.isChatScrolledToBottom) {
-				this.scrollToBottom()
-			}
-		},
-
-		/**
-		 * Scrolls to the bottom of the list (to show reaction to the last message).
+		 * Scrolls to the bottom of the list.
+		 * @param options {object} Options for scrolling (smooth: boolean, force: boolean)
 		 */
-		scrollToBottomIfSticky() {
-			if (this.isSticky) {
-				this.scrollToBottom()
+		scrollToBottom(options = null) {
```

Comment:
```suggestion
		scrollToBottom(options = {}) {
```
and no more optional checks